MessageHandler.js

```diff
@@ -25,14 +27,24 @@ class MessageHandler{
             return;
         }
 
-        // we only accept messages that start with "$"
-        if(!message || message.length < 1 || message[0] !== "$"){
+        // If this message starts with the comment prefix (and one is set)
+        // ignore this message no matter what
+        if (this.commentPrefix.length > 0 && message.startsWith(this.commentPrefix)){
```

Comment:
null will already work to disable it, since no string starts with null. What I was worried about people setting it to a empty string and expecting it to disable the behaviour, but instead it would treat every message like a comment.

How about
```
if (typeof this.commentPrefix == "string" && this.commentPrefix.length > 0 && message.startsWith(this.commentPrefix))
```

Or if you think zero-length strings are not an issue then simply
```
if (typeof this.commentPrefix != "undefined" && message.startsWith(this.commentPrefix))
```